codecs/jxl/enc/jxl_enc.cpp

```diff
@@ -102,7 +91,7 @@ val encode(std::string image, int width, int height, JXLOptions options) {
   auto result = jxl::ConvertImage(
       jxl::Span<const uint8_t>(reinterpret_cast<const uint8_t*>(image.data()), image.size()), width,
       height, jxl::ColorEncoding::SRGB(/*is_gray=*/false), /*has_alpha=*/true,
-      /*alpha_is_premultiplied=*/false, /*bits_per_sample=*/8, JXL_LITTLE_ENDIAN,
+      /*alpha_is_premultiplied=*/false, /*bits_per_sample=*/8, /*endiannes=*/JXL_LITTLE_ENDIAN,
```

Comment:
The new comment seems a bit unnecessary - unlike raw values, enum name is fairly self-descriptive.

Reply:
I think clang-tidy will produce warnings if say an argument gets added in the middle - as this is a C enum value, it would otherwise be silently converted to mean something entirely different...

Reply:
Ideally, I'd change the API of `ConvertImage` to accept a `struct` with named options rather than a long lits of params anyway :) Having lists this long where one needs to remember or manually annotate each argument is usually a code smell.

But I suppose this is not the final API, so it's fine.